general/server/media/shows.md

```diff
@@ -13,6 +13,10 @@ The most common naming scheme for shows is categorizing the files by series and
         /Season 01
             Episode S01E01.mkv
             Episode S01E02.mkv
+        /Season 02
+            Episode S02E01.mkv
+            Episode S02E02.mkv
+            
     /Series
         Episode S01E01.mp4
         Episode S01E02.mp4
```

Comment:
you should probably add the same here as well

Reply:
Excuse me, I do not know in which line you said. After "Episode S01E02.mp4" ?

Reply:
This one has no seasons to show that is an option as well.

Reply:
```suggestion
        Episode S01E02.mp4
        Episode S02E01.mp4
        Episode S02E02.mp4
```
forgot suggestions on non-changed lines were a thing

Reply:
I removed the extra new line.

One question... The algoritm detect right diferent names in the folder level of seasons?
It could be useful explain that with the example

> Show
>         /Season 01
>             Episode S01E01.mkv
>             Episode S01E02.mkv
>         /Season 02
>             Episode S02E01.mkv
>             Episode S02E02.mkv
>         /S3
>             Episode S03E01.mkv
>             Episode S03E02.mkv